stellarphot/settings/tests/test_models.py

```diff
@@ -24,102 +23,111 @@
     SourceLocationSettings,
 )
 
-DEFAULT_APERTURE_SETTINGS = dict(radius=5, gap=10, annulus_width=15, fwhm=3.2)
+TEST_APERTURE_SETTINGS = dict(radius=5, gap=10, annulus_width=15, fwhm=3.2)
 
 TEST_CAMERA_VALUES = dict(
-    data_unit=u.adu,
-    gain=2.0 * u.electron / u.adu,
+    data_unit="adu",
+    gain="2.0 electron / adu",
     name="test camera",
-    read_noise=10 * u.electron,
-    dark_current=0.01 * u.electron / u.second,
-    pixel_scale=0.563 * u.arcsec / u.pix,
-    max_data_value=50000 * u.adu,
+    read_noise="10.0 electron",
+    dark_current="0.01 electron / s",
+    pixel_scale="0.563 arcsec / pix",
+    max_data_value="50000.0 adu",
 )
 
-DEFAULT_EXOPLANET_SETTINGS = dict(
-    epoch=Time(0, format="jd"),
-    period=0 * u.min,
+TEST_EXOPLANET_SETTINGS = dict(
+    epoch={
```

Comment:
So that is everything hidden in a `Time` object?  Wow...

Reply:
yeah, I was a little surprised...